src/Common/Config/ConfigProcessor.cpp

```diff
@@ -673,7 +673,9 @@ XMLDocumentPtr ConfigProcessor::processConfig(
     zkutil::ZooKeeperNodeCache * zk_node_cache,
     const zkutil::EventPtr & zk_changed_event)
 {
-    LOG_DEBUG(log, "Processing configuration file '{}'.", path);
+    const bool write_logs = is_config_changed;
```

Comment:
since it is used only within this function it'd be better to make it `processConfig`'s  argument